src/main/groovy/com/twcable/grabbit/server/services/impl/TreeTraverser.java

```diff
@@ -0,0 +1,344 @@
+package com.twcable.grabbit.server.services.impl;
```

Comment:
Hi @mzgubin I did not see info about this in your PR description. Could you please let us know the need to re-implement this instead of using the TreeTraverser from Jackrabbit commons as it was before?

Reply:
Well the reason was that it allowed me to log what the error was when it was traversing. By default it was set to ErrorHandler.IGNORE so the errors were just being swallowed up.